ios/FLRNOptionsAdapter.m

```diff
@@ -102,4 +116,13 @@ - (NSString * _Nullable)getStringValueFor:(NSString *)key fromDictionary:(NSDict
     return nil;
 }
 
+- (BOOL)valueIsValidFor:(NSString *)key fromDictionary:(NSDictionary *)dict {
+    NSArray *allKeys = dict.allKeys;
+    id value = dict[key];
+    if ([value isKindOfClass:[NSString class]]) {
+        return [allKeys containsObject:key] && (value != nil) && ![value isKindOfClass:[NSNull class]] && [value length] > 0;
```

Comment:
In this branch, you already know that `[value isKindOfClass:[NSString class]]` is `true`, so of course that `[value isKindOfClass:[NSNull class]]` will be `false`.